src/Generator.Bind/Generator.Bind.csproj

```diff
@@ -274,4 +274,12 @@
     <Folder Include="Specifications\Docs\" />
   </ItemGroup>
   <Import Project="..\..\build\StyleCop.props" />
+  <ItemGroup>
```

Comment:
Is this not in StyleCop.props?

Reply:
It is but running paket puts it here as well. Not sure why.

Reply:
It doesn't resolve the imports, I don't think.

Reply:
This seems to break the StyleCop disabled build as well. Will need to look into this some.